x/cronos/rpc/api.go

```diff
@@ -390,20 +386,18 @@ func (api *CronosAPI) ReplayBlock(blockNrOrHash rpctypes.BlockNumberOrHash, post
 	return receipts, nil
 }
 
-// getBlockNumber returns the BlockNumber from BlockNumberOrHash
-func (api *CronosAPI) getBlockNumber(blockNrOrHash rpctypes.BlockNumberOrHash) (rpctypes.BlockNumber, error) {
-	switch {
-	case blockNrOrHash.BlockHash == nil && blockNrOrHash.BlockNumber == nil:
-		return rpctypes.EthEarliestBlockNumber, fmt.Errorf("types BlockHash and BlockNumber cannot be both nil")
-	case blockNrOrHash.BlockHash != nil:
-		blockHeader, err := api.backend.HeaderByHash(*blockNrOrHash.BlockHash)
-		if err != nil {
-			return rpctypes.EthEarliestBlockNumber, err
+// getBlock returns the BlockNumber from BlockNumberOrHash
```

Comment:
// getBlock returns the Block from BlockNumberOrHash

Reply:
```suggestion
// getBlock returns the block from BlockNumberOrHash
```